gen-app-builder/data-store-status-checker/README.md

```diff
@@ -0,0 +1,15 @@
+# Gen App Builder Data Store Status Checker
+_Using Google Cloud Discovery Engine APIs for Enterprise Search and Conversational AI_
+
+---
+
+## What is a Data Store?
+A [Data Store](https://cloud.google.com/generative-ai-app-builder/docs/create-datastore-ingest) in Gen AI App Builder is a collection of websites or documents, both structured and unstructured, that can be indexed for search and retrieval actions.
+
+Data Stores are the fundamental building block behind [Enterprise Search](https://cloud.google.com/enterprise-search) and [Generative AI Agents](https://cloud.google.com/generative-ai-app-builder/docs/agent-usage).
```

Comment:
Suggest mentioning in the README and the notebook, that this notebook will only work for Data Stores created for chat apps and not for Data Stores created for search apps (I think?). At least when I used it for a search data store the notebook immediately returned "finished" and 0 documents indexed.

Reply:
I agree on moving the README to the notebook. Maybe remove the examples folder and just have the notebook in the `data-store-status` directory

Reply:
Ack.

- Dropped README file and moved to notebook
- Added "Limitations" section to denote this only works for `Chat` data store types (for the time being)
- moved the notebook into the root `data-store-status` dir